examples/07-transactions.ts

```diff
@@ -0,0 +1,20 @@
+import { DataUtils } from '../src';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+async function main(): Promise<void> {
+  // initializating Data service...
+  const dataService = new DataUtils();
```

Comment:
> The API key is optional for DataUtils. If the user provides an API key, it will be used that; however, if an API key is not provided, the default API key will be used instead.

I understand that. But examples are required to elucidate how the apiKey must be used in DataUtils. So please provide an example of how that can be used.

Reply:
The example has been modified to include passing the data API key within the **DataUtils**.